### PR TITLE
[zh] add content/zh/docs/tasks/debug/debug-application/_index.md

### PR DESCRIPTION
**Pull request by JarHMJ**

### CODE REVIEW

File: content/zh/docs/tasks/debug/debug-application/_index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: "应用故障排除"
+description: 调试常见的容器应用问题.
+weight: 20
+--- 
+
+<!-- 
+---
```
Comment (tengqm) [inline content/zh/docs/tasks/debug/debug-application/_index.md:8]: remove this line.

File: content/zh/docs/tasks/debug/debug-application/_index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+title: "Troubleshooting Applications"
+description: Debugging common containerized application issues.
+weight: 20
+---
```
Comment (tengqm) [inline content/zh/docs/tasks/debug/debug-application/_index.md:12]: remove this line.

File: content/zh/docs/tasks/debug/debug-application/_index.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+title: "Troubleshooting Applications"
+description: Debugging common containerized application issues.
+weight: 20
+--- 
+-->
+
+<!-- 
+This doc contains a set of resources for fixing issues with containerized applications. It covers things like common issues with Kubernetes resources (like Pods, Services, or StatefulSets), advice on making sense of container termination messages, and ways to debug running containers.
+-->
+
```
Comment (tengqm) [inline content/zh/docs/tasks/debug/debug-application/_index.md:18]: remove this line.

File: content/zh/docs/tasks/debug/debug-application/_index.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+该文档包含了一些用来解决容器应用问题的资源。它涵盖了 Kubernetes 资源（如 Pod、Service 或 StatefulSets）的常见问题，理解容器终止消息的建议以及调试正在运行的容器的方法。
```
Comment (tengqm) [inline content/zh/docs/tasks/debug/debug-application/_index.md:19]:
```suggestion
此文档包含了一些用来解决容器应用问题的资源。它涵盖了 Kubernetes 资源（如 Pod、Service 或 StatefulSets）
的常见问题，理解容器终止消息的建议以及调试正在运行的容器的方法。
```

File: content/zh/docs/tasks/debug/debug-application/_index.md (hunk 2 above)
Comment (JarHMJ) [reply content/zh/docs/tasks/debug/debug-application/_index.md]: done

**Comment by JarHMJ** [comment]
done

**Pull request merged**